cmd/gaia/cli_test/test_helpers.go

```diff
@@ -56,17 +56,16 @@ type Fixtures struct {
 
 // NewFixtures creates a new instance of Fixtures with many vars set
 func NewFixtures(t *testing.T) *Fixtures {
-	tmpDir := os.TempDir()
-	gaiadHome := fmt.Sprintf("%s%s%s%s.test_gaiad", tmpDir, string(os.PathSeparator), t.Name(), string(os.PathSeparator))
-	gaiacliHome := fmt.Sprintf("%s%s%s%s.test_gaiacli", tmpDir, string(os.PathSeparator), t.Name(), string(os.PathSeparator))
+	tmpDir, err := ioutil.TempDir("", "gaia_integration_"+t.Name()+"_")
```

Comment:
Should we not use `TempDir()` instead of `""` here?

Reply:
From https://golang.org/pkg/io/ioutil/#TempDir

>  If dir is the empty string, TempDir uses the default directory for temporary files (see os.TempDir)